debian/control

```diff
@@ -0,0 +1,19 @@
+Source: supervision-scripts
+Section: admin
+Priority: optional
+Maintainer: Luke Diamand <luke@diamand.org>
+Build-Depends: debhelper (>= 9)
+Standards-Version: 3.9.5
+Homepage: https://github.com/apayne/supervision-scripts.git
```

Comment:
FYI, Github is actually just a mirror, but that's ok.  The main repo is on bitbucket.